Update content to change language to client-facing, update quickstart (#1125)
novuhq/docs: content/docs/agents/managed-agent/quickstart.mdx

---
title: "Quickstart"
pageTitle: 'Managed Agent Quickstart'
description: 'Use the Novu CLI to create a managed agent, connect it to Slack, and get a reply in minutes.'
icon: Zap
---

import { Brain, Link, MessagesSquare } from 'lucide-react';

This guide walks you through creating a managed agent using the Novu CLI, connecting it to Slack, and sending your first message.

![Novu connect demo](/images/agents/quickstart/novu-connect-demo.gif)

## Prerequisites

* Node.js installed on your machine.
* A [Novu account](https://connect.novu.co/) (you can create one during the setup flow).
* A Slack workspace where you can install apps.

<Steps>

<Step>

## Run the CLI

In your terminal, run:

```bash
npx novu connect
```

</Step>

<Step>

## Create an agent

> 

Reviewer: Add context paragraph?

> 1. Choose where your agent runs. The CLI asks **Where do you want the agent to run?** Select an option and press **Enter**:

Reviewer: ```suggestion
1. Choose where your agent runs. The CLI asks **Where do you want the agent to run?** , select an option and then press **Enter**:
```

    * **Demo credentials**: The fastest way to start. No Anthropic API key required (Novu handles it), with up to 10 conversations per month.

>     * **Claude Managed Agents**: Bring your own Anthropic API key.

Reviewer: I would leave out the explanation of all the options. Might just confuse the user.

    * **AWS Claude Managed Agents**: Use your AWS Claude credentials (API key, region, and workspace ID).
    For this quickstart, select **Demo credentials**.
2. Describe your agent. Type a description of what you want the agent to do and press **Enter**.

>     ```text

Reviewer: So, do you want them to enter a description? You do not explicitly tell them to enter the text below as a description.

    Create an engineering assistant agent that checks Linear and GitHub for open issues, pending PR reviews, and review requests. It summarizes what needs attention, flags blockers and stale tickets, and posts a short daily standup digest. It keeps responses concise and links back to the relevant Linear issue or GitHub pull request.
    ```
    Novu will generate the agent for you, including its system prompt, tools, MCP servers, and skills based on your description.
3. Review and create the agent. The CLI shows the generated agent for you to review. You can adjust any of the following fields before confirming:
      * Name
      * Identifier
      * System prompt
      * Tools
      * MCP servers
      * Skills
      

>   If everything looks good, select **Create this agent** and press **Enter**.

Reviewer: ```suggestion
  If everything looks good, then select **Create this agent** and press **Enter**.
```

>   If you want to change the description and regenerate, select **Regenerate from description** and repeat the previous step.

Reviewer: ```suggestion
  If you want to change the description and regenerate, then select **Regenerate from description** and repeat the previous step.
```

</Step>

<Step>

## Connect Slack

The CLI prompts you: **pick a channel to connect this agent to.** Select **Slack** and press **Enter**.
 

> Novu will ask you to paste a Slack App Configuration Token. To get one:

Reviewer: ```suggestion
Novu asks you to paste a Slack App Configuration Token. To get one:
```

1. Open [https://api.slack.com/apps](https://api.slack.com/apps).
2. Scroll to the bottom of the page.
3. Under **App Configuration Tokens**, click **Generate Token**.
4. Copy the access token, it starts with `xoxe.xoxp-`.
5. Paste the token into the terminal and press **Enter**.
    <Callout type="info">
    The CLI sends the token to your Novu account once to create the Slack app, then discards it; the token isn't stored.
    </Callout>
6. Add the app to your workspace. After pasting the token, the CLI prompts you to add the app to your Slack workspace. Press **Enter** and follow the link to install the bot.

Once installed, your agent is live and listening for messages in Slack.

</Step>

<Step>

## Send a message

> Open Slack, where you'll see a message from your agent. Send it a direct message to get started. You'll then be prompted by the agent to connect your Linear and GitHub MCP servers, once you do, your agent is ready.

Reviewer: Rewrite without "will" wherever possible. Apply throughout.
